lib/Fetcher/Client/FeedIoClient.php

```diff
@@ -47,7 +47,7 @@ public function getResponse(string $url, \DateTime $modifiedSince) : ResponseInt
         try {
             $options = [
                 'headers' => [
-                    'If-Modified-Since' => $modifiedSince->format(\DateTime::RFC2822)
+                    'If-Modified-Since' => $modifiedSince->format('D, d M Y H:i:s \G\M\T')
```

Comment:
I just checked but I think this misrepresents the time, a fix would be to do this before:
```php
$modifiedSince->setTimezone(new DateTimeZone('UTC'));
```

Reply:
I agree, or rather

```
$modifiedSince->setTimezone(new DateTimeZone('GMT'));
```
and
```
'If-Modified-Since' => $modifiedSince->format('D, d M Y H:i:s e')
```

Reply:
`$modifiedSince->setTimezone(new \DateTimeZone('UTC'));`
`$modifiedSince->format('D, d M Y H:i:s e')`

Both.

Reply:
But UTC will cause BAD REQUEST as the standard says to use GMT

Reply:
Use GMT then ;) Seems to be a valid timezone even if UTC is preferred nowdays: https://3v4l.org/N4M8X

Reply:
I'm only following the RFCs :)